.github/workflows/contract-testing.yml

```
@@ -0,0 +1,271 @@
+          # Build schemathesis command
+          COMMAND="schemathesis run openapi.json \
+            --base-url $BASE_URL \
+            --hypothesis-max-examples $MAX_EXAMPLES \
+            --hypothesis-deadline 10000 \
+            --workers 3 \
+            --request-timeout 60 \
+            --report-format json \
+            --output-file schemathesis-${{ matrix.test-suite.name }}-report.json \
+            --validate-schema true \
+            --checks ${{ matrix.test-suite.checks }}"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)